reverse_engineering/helpers/fetchRequestHelper.js

```diff
@@ -81,19 +83,74 @@ const destroyActiveContext = () => {
 	return result;
 };
 
-const fetchApplyToInstance = async (connectionInfo, logger) => {
+/**
+ * @return {(
+ *     connectionInfo: Object,
+ *     samples: Array<Object>,
+ *     entityJsonSchema: Object,
+ * ) => Promise<any>}
+ * */
+const sendSampleBatch = (_) => (connectionInfo, samples, entityJsonSchema) => {
+	const script = generateSamplesScript(_)(entityJsonSchema, samples);
+	return executeCommand(connectionInfo, script, 'sql')
+}
+
+/**
+ * @param lineNumber {number}
+ * @return {boolean}
+ * */
+const shouldLogStep = (lineNumber) => {
+	if (lineNumber < 1000) {
+		return lineNumber % 100 === 0;
+	}
+	return lineNumber % 1000 === 0;
+}
+
+
+const sendSampleBatches = (_, logger) => async (connectionInfo) => {
+	const { entitiesData } = connectionInfo;
+	const batchSize = 5000;
```

Comment:
Maybe it is better to move `batchSize` to module level constant and use one value for batch insert and logging (line 103)

Reply:
I made `batchSize` a module-level constant but the constant for logging has a completely different meaning and purpose, so it is not a good idea to reuse this value there